package.json

```diff
@@ -28,7 +30,8 @@
     "colors": "^1.3.2",
     "express": "^4.16.3",
     "json-web-key": "^0.3.0",
-    "jsonwebtoken": "^8.3.0"
+    "jsonwebtoken": "^8.3.0",
+    "snyk": "^1.161.1"
```

Comment:
Should this be a dependency or a dev dependency? Is it actually used at runtime, or only as part of dev/ci?

Reply:
Ah, nice pickup. This is definitely a mistake. Thanks!

Reply:
Fixed in f2f8098

Reply:
Thanks!